ferrocene/doc/internal-procedures/src/subtree-pulls.rst

```diff
@@ -78,7 +78,7 @@ Now run::
 The script will attempt to update all the registered subtrees in order
 until all succeed or the first one fails.
 
-You can also pass the name of a repository to only pull that specific repo::
+You can also pass the name of a repository to only pull that specific repo:
 
   ferrocene/tools/pull-subtrees/pull.py <SUBTREE_REPO>
```

Comment:
Ok, so the problem here is that the `::` need the next paragraph to be indented by three spaces, not by two spaces.

```suggestion
You can also pass the name of a repository to only pull that specific repo::

   ferrocene/tools/pull-subtrees/pull.py <SUBTREE_REPO>
```

Reply:
We should certify a whitespace compiler.